ModelicaTest/Tables/CombiTable2Ds.mo

```diff
@@ -535,7 +535,7 @@ double mydummyfunc(double dummy_in) {
     Modelica.Blocks.Tables.CombiTable2Ds combiTable2DT(
       table=transpose([1,0,6;2,4,4;3,9,9;4,16,16]),
       smoothness=Modelica.Blocks.Types.Smoothness.ConstantSegments) annotation(Placement(transformation(extent={{-95,-60},{-75,-40}})));
-    equation
+  equation
       connect(combiTable1D.u,clock1.y) annotation(Line(
```

Comment:
The indentation rule shall be in fact applied also on "connect(..." too. But I see no necessity to fix this in ModelicaTest library.

Reply:
Agreed, so wait with that.